Shell/LineEditor.cpp

```diff
@@ -106,15 +111,16 @@ void LineEditor::cut_mismatching_chars(String& completion, const String& other,
     completion = completion.substring(0, i);
 }
 
-void LineEditor::tab_complete_first_token(const String& token)
+Vector<String> LineEditor::tab_complete_first_token(const String& token)
 {
     auto match = binary_search(m_path.data(), m_path.size(), token, [](const String& token, const String& program) -> int {
         return strncmp(token.characters(), program.characters(), token.length());
     });
     if (!match)
-        return;
+        return Vector<String>();
 
```

Comment:
I believe it's more idiomatic to write this as `return {};`

Reply:
Heh, I didn't know that was valid syntax. Cool.